code/__DEFINES/reactions.dm

```diff
@@ -44,10 +44,10 @@
 #define STIMULUM_SECOND_RISE				0.0009
 #define STIMULUM_ABSOLUTE_DROP				0.00000335
 #define REACTION_OPPRESSION_THRESHOLD		5
-#define NOBLIUM_FORMATION_ENERGY			2e9 	//1 Mole of Noblium takes the planck energy to condense.
+#define NOBLIUM_FORMATION_ENERGY			2e7
```

Comment:
I'm kinda uncomfortable with making nobilium easier to make, due to bombs, the explosive reaction I mentioned earlier, but even with that, wouldn't this energy  value cause an insane amount of heat on first react? it's 4000x the heat capacity even now

Reply:
i tested it and with a couple of t3 freezers you can easily handle it; nob bombs are still difficult to make because even tho this PR simplify the making of it, you still have to produce the trit, and have the t3 freezers in hand, also for the bombs you'd need stim to be effective.